api/requirements.txt

```diff
@@ -67,6 +67,7 @@ kombu==4.6.7
 lxml==4.5.0
 MarkupSafe==1.1.1
 mccabe==0.6.1
+mistune==2.0.0a6
```

Comment:
Please add mistune to the `requirements/req-base.txt` without a set version

Reply:
👍  added this